consensus/src/alerts/mod.rs

```diff
@@ -106,7 +106,7 @@ pub enum AlertMessage<H: Hasher, D: Data, S: Signature, MS: PartialMultisignatur
     /// Alert regarding forks, signed by the person claiming misconduct.
     ForkAlert(UncheckedSigned<Alert<H, D, S>, S>),
     /// An internal RMC message, together with the id of the sender.
-    RmcMessage(NodeIndex, RmcMessage<H::Hash, S, MS>),
+    RmcHash(NodeIndex, RmcMessage<H::Hash, S, MS>),
```

Comment:
Probably un-change this name, now it's just confusing. :P

Reply:
good catch